sphinxcontrib/spelling/__init__.py

```diff
@@ -43,4 +47,4 @@ def setup(app):
     app.add_config_value('spelling_ignore_importable_modules', True, 'env')
     # Add any user-defined filter classes
     app.add_config_value('spelling_filters', [], 'env')
-    return {"parallel_read_safe": False}
+    return {"parallel_read_safe": True}
```

Comment:
It'd be useful to explicitly declare the parallel write support level and the version:
```suggestion
    return {
        'parallel_read_safe': True,
        'parallel_write_safe': True,
        'version': __version__,
    }
```

(import the version somewhere on top of read via `importlib.metadata`/`pkg_resources`)

Reply:
Good idea. Done.